<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>glossary::gui::view::DictionariesTab</class>
    <widget class="QWidget" name="glossary::gui::view::DictionariesTab">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>592</width>
                <height>502</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <widget class="QListView" name="listOfDictionaries">
            <property name="enabled">
                <bool>true</bool>
            </property>
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>40</y>
                    <width>151</width>
                    <height>301</height>
                </rect>
            </property>
            <property name="modelColumn">
                <number>0</number>
            </property>
        </widget>
        <widget class="QLabel" name="labelDictionaries">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>20</y>
                    <width>101</width>
                    <height>16</height>
                </rect>
            </property>
            <property name="text">
                <string>Dictionaries:</string>
            </property>
        </widget>
        <widget class="QLabel" name="labelDictionaryWords">
            <property name="geometry">
                <rect>
                    <x>270</x>
                    <y>10</y>
                    <width>131</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>Dictionary words:</string>
            </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
                <rect>
                    <x>450</x>
                    <y>120</y>
                    <width>121</width>
                    <height>91</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QPushButton" name="buttonAddWord">
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Add word</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="buttonRemoveWord">
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Remove word</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="buttonUpdateTranslation">
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Update translation</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>350</y>
                    <width>152</width>
                    <height>118</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <item>
                            <widget class="QPushButton" name="buttonAddNewDictionary">
                                <property name="font">
                                    <font>
                                        <pointsize>10</pointsize>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>Add new dictionary</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="buttonAddDictionaryFromFile">
                                <property name="font">
                                    <font>
                                        <pointsize>10</pointsize>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>Add dictionary from file</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="buttonRemoveDictionary">
                                <property name="font">
                                    <font>
                                        <pointsize>10</pointsize>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>Remove dictionary</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QPushButton" name="buttonUpdateTranslations">
                        <property name="font">
                            <font>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>Update translations</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QListView" name="listOfDictionaryWords">
            <property name="geometry">
                <rect>
                    <x>230</x>
                    <y>40</y>
                    <width>211</width>
                    <height>431</height>
                </rect>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
